cmd/infra/powervs/destroy.go

```diff
@@ -99,6 +103,9 @@ func NewDestroyCommand() *cobra.Command {
 	cmd.Flags().StringVar(&opts.CloudConnection, "cloud-connection", opts.CloudConnection, "IBM Cloud PowerVS Cloud Connection. Use this flag to reuse an existing Cloud Connection resource for cluster's infra")
 	cmd.Flags().StringVar(&opts.CloudInstanceID, "cloud-instance-id", opts.CloudInstanceID, "IBM PowerVS Cloud Instance ID. Use this flag to reuse an existing PowerVS Cloud Instance resource for cluster's infra")
 	cmd.Flags().BoolVar(&opts.Debug, "debug", opts.Debug, "Enabling this will result in debug logs will be printed")
+	cmd.Flags().BoolVar(&opts.PER, "power-edge-router", opts.PER, "Enabling this flag will denotes that Power Edge Router solution is used while creating the cluster")
```

Comment:
The description doesn't seem to make sense for the destroy command. Something like, "Enabling this flag ensures that the Power Edge router that was used to create the cluster is cleaned up."